docker-compose.yml

```diff
@@ -10,6 +10,10 @@ services:
     volumes:
       - ./priv/wal.sql:/docker-entrypoint-initdb.d/wal.sql
       - pg-data:/var/lib/postgresql/data
+  stripe-mock:
```

Comment:
Would relying on this affect tests?

An alternative approach is to have a mock cowboy server and add it to the application tree on test env.

Using this for dev env is fine as we want it as close to realistic as possible, but if this was the consideration then it makes more sense to just use the actual stripe api

Reply:
It won't affect tests since we're only setting the config for dev.

I wanted to avoid complex solutions and have something quick that is supported by Stripe.

Using it for dev only at the moment and it works for the main scenario which is "accessing the billing tab without errors". If we need more complex examples that require the real API then we can override with `runtime.exs` the base url.

Reply:
ok sounds good 👍